src/pages/includes/wallet-info.ts

```diff
@@ -0,0 +1,18 @@
+import { Component, Input } from "@angular/core";
+
+@Component({
+  selector: 'page-wallet-info',
+  templateUrl: 'wallet-info.html',
+})
+export class WalletInfoPage {
+  public _wallet: any;
```

Comment:
If this variable is not used in the view, it should be private. I think the '_' is unnecessary due to typing

Reply:
Changed to private.
The underscore is needed to avoid conflicts with the method names.